azure/aks/example-usage/README.md

```diff
@@ -0,0 +1,67 @@
+# Example usage of Azure Terraform modules
```

Comment:
`diagram.jpg` it's a little bit messy IMO. I mean, when you use a diagram after a first look you should know what is happening. On this diagram, you have to analyze the diagram.

Reply:
Will try to do sth about it